test/testSetup.ts

```diff
@@ -15,16 +15,13 @@ import {
   QuickOpenBox,
   Workbench
 } from 'wdio-vscode-service';
-import {
-  EnvironmentSettings
-} from './environmentSettings';
+import { EnvironmentSettings } from './environmentSettings';
```

Comment:
All the changes in this file seem to be spacing changes. Is your editor using a abiding by the `.prettierrc`?